### DISCUSSION TITLE
.NET Maui WinUI fails to deploy after adding the WinUIEx and Microsoft.WindowsAppSDK packages

### DISCUSSION THREAD

**Discussion by ataparia** [dotMorten/WinUIEx#139] [General]
.NET Maui WinUI app fails to deploy after adding the WinUIEx and Microsoft.WindowsAppSDK packages with the following message:

AppxManifest.xml(29,27): error 0x80070002: Cannot install or update package maui-package-name-placeholder_x521x066and8w because the splash screen image [$placeholder$.png] cannot be located. Verify that the package contains an image that can be used as a splash screen for the application, and that the package manifest points to the correct location in the package where this splash screen image can be found.

Same app deploys correctly if I remove the following lines of code from my csproj file:
PackageReference Include="WinUIEx" Version="2.3.0" Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'"
PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230822000" Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'"

**Comment by dotMorten** [comment]
Is it caused by the first or the second package reference? It's highly unlikely this is caused by WinUIEx, and more likely it's caused by WindowsAppSDK

**Reply by dotMorten** [reply]
Also see https://github.com/dotnet/maui/issues/4736

**Reply by ataparia** [reply]
You are absolutely right. Seems like its caused by the Microsoft.WindowsAppSDK nuget package reference. Is there any way around it as it seems like I am forced to use this nuget package in order to reference WinUIEx package.

**Reply by dotMorten** [reply]
Your MAUI project is out of date - see linked issue and google the error. Lots of references to the issue.

**Reply by ataparia** [reply]
Its strange though that removing those nuget package references get's me passed the issue described. Having said that, I will see what I need to do in order to update my maui bits. Thanks!

**Comment by mdmontesinos** [comment]
I'm also getting the same error just after adding WinUIEx and Microsoft.WindowsAppSDK packages, in an attempt to solve WebAuthenticator problem in Windows.

I can confirm that my MAUI Blazor project is up-to-date (version 7.0.92).

Also, to be sure, I just created a fresh project with MAUI Blazor template, compiled and checked that it works. Added package references according to guidelines:

```
<PackageReference Include="WinUIEx" Version="2.3.0" Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'" />
<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230822000" Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'" />
```

And now I get the same crash as @ataparia 

EDIT: I just updated to newest MAUI version, 7.0.96, that was released yesterday, and the error does not occur anymore.

**Reply by ataparia** [reply]
Thanks @mdmontesinos . I can also confirm that this thing is working fine after updating my VS 2022 to the latest version.
